incubator/sparkoperator/README.md

```diff
@@ -34,8 +34,7 @@ The following table lists the configurable parameters of the Spark operator char
 | `controllerThreads`       | Number of worker threads used by the SparkApplication controller | 10                                 |
 | `ingressUrlFormat`        | Ingress URL format                                           | ""                                     |
 | `logLevel`                | Logging verbosity level                                      | 2                                      |
-| `installCrds`             | Whether to install CRDs                                      | true                                   |
-| `cleanupCrdsBeforeInstall` | Remove CRDs before running the crd-install hook on changes. | `false` |
+| `installCrds`             | Whether to install CRDs                                      | false                                   |
```

Comment:
One more thing - we should probably add a similar note to all relevant charts - there's a great note added here https://github.com/helm/charts/pull/19287

> Whether the release should install CRDs. Regardless of this value, Helm v3+ will install the CRDs if those are not present already. Use `--skip-crds` with `helm install` if you want to skip CRD creation

Would you also please add something to this effect, since this PR is still open?

Reply:
good point, done.